iommi/table__tests.py

```diff
@@ -3745,3 +3745,66 @@ def test_auto_rowspan_twice():
     table = Table(rows=[Struct(a=1), Struct(a=1)], columns__a__auto_rowspan=True).refine_done()
     table.bind(request=req('get')).__html__()
     table.bind(request=req('get')).__html__()
+
+
+@pytest.mark.django_db
+def test_pagination_with_thousands_separator(settings):
+    settings.USE_THOUSAND_SEPARATOR = True
+    settings.USE_L10N = True
+
+    for x in range(1_002):
+        TFoo(a=x, b="foo").save()
+
+    class TestTable(Table):
+        a = Column.number()
+
+    verify_table_html(
+        find=dict(name='nav'),
+        table=TestTable(rows=TFoo.objects.all().order_by('pk')),
+        query={'page_size': 1, 'page': 10001},
+        expected_html="""
+        <nav aria-label="Pages">
+            <ul>
+                <li>
+                    <a href="?page_size=1&amp;page=1" aria-label="First Page">&laquo;</a>
+                </li>
+            
```

Comment:
No need for the empty lines

Reply:
True. And could be more compact otherwise too.